test/fixtures/cookbooks/rack_user_test/recipes/remove.rb

```diff
@@ -1,5 +1,10 @@
 # comments!
 
+user 'rack' do
```

Comment:
That's being tested in the create.rb and create_with_password.rb. The remove.rb was an existing test that wasn't actually testing anything, but the way the tests are segregated implies that it should remove an already existing user. This user can be created in any way possible, not limited to `rack_user`.